### DISCUSSION TITLE
How do I set unique permissions on a site page

### DISCUSSION THREAD

**Discussion by metkhoo** [pnp/powershell#1405] [Q&A]
Hi all,

I can see there is **Set-PnPWebPermission** for Webs (sites), **Set-PnPFolderPermission** for folders in document libraries and Set-**PnPListItemPermission** for custom list items.
But where is the command to set unique permissions on a specific file in a document library or even a specific page in SitePages library?

I can't see a 
**Set-PnPFilePermission** command.
Is there another way to set unique permissions on a specific page in a SitePages library?

Thanks

**Comment by LeonArmston** [comment]
Hi 

I've just answered this in the Issue you raised https://github.com/pnp/powershell/issues/1406

Let me know if this helps?

Files you would use the same approach.
